src/api/teams/helpers/github/github-shared-repo-access.js

```diff
@@ -0,0 +1,54 @@
+import { config } from '~/src/config'
+import { createLogger } from '~/src/helpers/logging/logger'
+
+const logger = createLogger()
+
+// returns 204 response if team already has access
+const addSharedRepoAccess = async (octokit, githubTeam) => {
+  const orgName = config.get('gitHubOrg')
+  const sharedRepos = config.get('sharedRepos')
+
+  const p = sharedRepos.map((repo) => {
+    logger.info(`Granting ${githubTeam} access to ${orgName}/${repo}`)
```

Comment:
Pretty sure this will log when the map is iterated, rather than when the promise is called

Reply:
Still relevant